### PR TITLE
add chainmanager readme

### PR DESCRIPTION
**Pull request by temaniarpit27**
# Description

Add readme for chainmanager module

### CODE REVIEW

**Review by anshalshukla — APPROVED** [review]

**Review by pratikspatil024 — APPROVED** [review]

File: chainmanager/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Chainmanager Module
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Query commands](#query-commands)
+
+## Overview
+
+The chainmanager module is responsible for fetching the chainmanager params. These params include contract address of mainchain and maticchain, chain ids, mainchain and maticchain confirmation blocks
```
Comment (Raneet10) [inline chainmanager/README.md:10]: Just to be clear, should we elaborate a bit more on what `mainchain` and `maticchain` mean here ?

Comment (temaniarpit27) [reply chainmanager/README.md]: updated. Please check

Comment (Raneet10) [reply chainmanager/README.md]: Also a note for the future, we should probably replace the term `maticchain` with `borchain` or something. Thoughts @maticnetwork/pos-v1 ?

Comment (Raneet10) [reply chainmanager/README.md]: > updated. Please check
LGTM , thanks!

Comment (temaniarpit27) [reply chainmanager/README.md]: @Raneet10 we can continue this discussion on slack for better responses. Merging this for now

Comment (Raneet10) [reply chainmanager/README.md]: Sure, sure.

**Pull request merged**